consensus/polybft/sc_integration_test.go

```diff
@@ -296,7 +296,7 @@ func TestIntegration_DistributeFee(t *testing.T) {
 
```

Comment:
distribute dao incentive is not executed in the integration tests. Also, ensure proper results are available after the execution.